css/properties/white-space.json

```
@@ -96,6 +96,54 @@
             }
           }
         },
+        "nowrap": {
+          "__compat": {
+            "description": "<code>nowrap</code>",
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": "18"
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "3.5"
+              },
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "8"
```

It looks like this showed up a bit earlier too. For example, some bug reports:

https://www.quirksmode.org/bugreports/archives/2005/02/White_space_IE55_bug.html
https://www.quirksmode.org/bugreports/archives/2005/07/Wrapped_inlinenowrap_elements_lose_padding_left_in.html

I think it's possible it showed up even earlier, but I wasn't able to find any mentions of `nowrap` that wasn't the HTML attribute for IEs before version 5.5. This is also the version for the property as a whole (the parent feature). So let's go with that:

```suggestion
                "version_added": "5.5"
```